src/sign.builds

```diff
@@ -32,12 +32,14 @@
     <ReadSigningRequired MarkerFiles="@(SignMarkerFile)">
       <Output TaskParameter="SigningMetadata" ItemName="FilesToSign" />
     </ReadSigningRequired>
+    <Message Text="Attempting to sign %(FilesToSign.Identity) with authenticode %(FilesToSign.Authenticode) and strongname %(FilesToSign.StrongName)" />
```

Comment:
I think so as, it will only show up in our official builds and it would be nice to have that information readily available in our build logs.